onnxruntime/test/providers/cpu/reduction/reduction_ops_test.cc

```diff
@@ -3373,6 +3373,241 @@ TEST(ReductionOpTest, ReduceSum_ReduceDimWithZero3) {
   run(test3);
 }
 
+// test if noop_with_empty_axes behaves correctly
+TEST(ReductionOpTest, ReduceL1_noop_axes_input_initializer_opset_18) {
+  OpTester test("ReduceL1", 18);
+  test.AddAttribute("keepdims", (int64_t)0);
+  test.AddAttribute("noop_with_empty_axes", (int64_t)1);
+  test.AddInput<float>("data", {1, 2, 2},
+                       {1.0f, 2.0f,
+                        3.0f, 4.0f});
+  test.AddInput<int64_t>("axes", {0}, {}, true);
+  test.AddOutput<float>("reduced", {1, 2, 2}, {1.0f, 2.0f, 3.0f, 4.0f});
+  test.Run(
+      OpTester::ExpectResult::kExpectSuccess,
+      "",
+      {kTensorrtExecutionProvider,
+       kOpenVINOExecutionProvider,
+       kDnnlExecutionProvider,
+       kDmlExecutionProvider});
```

Comment:
@jywu-msft @fdwr
FYI about some newly added tests which have been excluded for these EPs.
Taking a guess at the point of contact for these EPs. If it's someone else, please let them know.

Reply:
@linnealovespie I think you added `noop_axes` recently to the DML EP reduction operators, right?